wg-manifests/charter.md

```diff
@@ -1,70 +1,32 @@
-# WG Manifests Charter
+# WG Platform/Manifests Charter
```

Comment:
@juliusvonkohout If you want to update the Charter of this WG, can you follow this template ?

https://github.com/kubeflow/community/blob/a5401ba5362e05850f2029b9e461fbeb663c93af/wgs/templates/wg-charter-template.md

Reply:
Yes, i will try to adjust it.

Reply:
done